src/ziggurat/messaging/rabbitmq_cluster_wrapper.clj

```diff
@@ -0,0 +1,73 @@
+(ns ziggurat.messaging.rabbitmq-cluster-wrapper
+  (:require [ziggurat.messaging.rabbitmq.cluster.connection :as rmq-cluster-conn]
+            [ziggurat.messaging.rabbitmq.cluster.producer :as rmqc-producer]
+            [ziggurat.messaging.messaging-interface :refer [MessagingProtocol]]
+            [ziggurat.messaging.rabbitmq.consumer :as rmq-consumer]))
+
+(def connection (atom nil))
+
+(def rabbitmq-cluster-config (atom nil))
```

Comment:
why store it in an atom? why not a fn that fetches it from config?

Reply:
that function will refer to Ziggurat config which will create a dependency on config ns inside the wrapper ns